coordinator/src/main/scala/com/socrata/datacoordinator/secondary/SecondaryWatcher.scala

```diff
@@ -47,17 +53,26 @@ class SecondaryWatcher[CT, CV](universe: => Managed[SecondaryWatcher.UniverseTyp
         "endingDataVersion" -> job.endingDataVersion
       ) {
         log.info(">> Syncing {} into {}", job.datasetId, secondary.storeId)
+        if (job.retryNum > 0) log.info("Retry #{} of {}", job.retryNum, maxRetries)
         try {
           playbackToSecondary(secondary, job)
           log.info("<< Sync done for {} into {}", job.datasetId, secondary.storeId)
         } catch {
           case e: Exception =>
-            log.error("Unexpected exception while updating dataset {} in secondary {}; marking it as broken",
-                      job.datasetId.asInstanceOf[AnyRef], secondary.storeId, e)
-            secondaryManifest.markSecondaryDatasetBroken(job)
+            if (job.retryNum < maxRetries) {
```

Comment:
This pr sets broken after a number of retries, but it seems reasonable to set broken after the first failure.

And with a retry backoff of 1 minute and exponential growth of 2^n sure it will take that long, but how often do we want to retry and how quickly to give up? With the proposed values it will be done retrying after only 14 minutes.